temp.txt

```diff
@@ -0,0 +1,339 @@
+Angleterre
```

Comment:
Is it expected that these two files temp.txt and temp2.txt are included?

Reply:
Oh no ! I just missclicked in vscode and push everything, I was thinking that I managed to clean my mistake, but apparently not :/